### ISSUE TITLE
Daydream Mirage Blank Screen

### ISSUE THREAD

**Issue by XAPHYRA** [googlevr/gvr-unity-sdk#1043]
### SPECIFIC ISSUE ENCOUNTERED
When I create a new project from scratch using the demo scene "HelloVR", It runs smoothly in the unity editor, but after building with 0 errors on console; There is just a blank screen on the headset. 

### HARDWARE/SOFTWARE VERSIONS
- Unity: 2019.10f1
- Google VR SDK for Unity: 1.200.0
- Device manufacturer, model, and O/S: Lenovo Mirage Solo

### ADDITIONAL COMMENTS
I've been able to run multiple self-made apps on the same device with no problems with previous Unity and SDK versions, so probably the problem is related with either of the latest released versions.

**Comment by XAPHYRA** [comment]
Yep; confirmed, 2019 unity versions together with the 1.200.0 SDK are the problem. I downgraded my unity project to 2018.3.14f1 and everything is fine now. YAY!!

**Comment by rusmaxham** [comment]
Can you ensure your 2019 project has the *Google VR Android* package installed?  

Window Menu > Package Manager > Google VR Android

**Comment by XAPHYRA** [comment]
Yes, the package is installed. Also I recently updated to unity 2019.1.11f1 to see if the issue has been fixed, but now; the app just quits as soon as I try to open it.
